recipes-devtools/cppmicroservices/cppmicroservices_4.0.bb

```diff
@@ -0,0 +1,63 @@
+inherit cmake
+
+SUMMARY = "C++ Micro Services"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
+
+# Packages required to build
+DEPENDS_append_class-target += "cppmicroservices-native"
+
+# Parallel make does not work with some recipes
+PARALLEL_MAKE = ""
```

Comment:
this is gonna to slow build , are we sure we dont want to make parallel build ?

Reply:
This was ported from SGX recipe.